### PR TITLE
setup webpack

### PR DESCRIPTION
**Pull request by clfhhc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0",
+    "webpack": "^4.10.2",
+    "webpack-cli": "^2.1.5"
+  },
+  "version": "1.0.0",
+  "main": "server/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/n-app/reviews.git"
```
Comment (rtsuzaki) [inline package.json]: should the url have "git+" in the front? I'm not sure. Please double check

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: {main: "./client/src/js/index.js"},
```
Comment (rtsuzaki) [inline webpack.config.js]: It'd be better to add __dirname in front of filepath

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
```
Comment (rtsuzaki) [inline webpack.config.js]: inconsistent between using single quotes or double quotes

File: webpack.config.js (hunk 2 above)
Comment (clfhhc) [reply webpack.config.js]: One of the tutorial use relative path

**Pull request closed without merging**